src/utils/gpu-filter-utils.js

```diff
@@ -92,13 +92,10 @@ export function resetFilterGpuMode(filters) {
 
 /**
  * Initial filter uniform
- * @returns {{filterMin: Array<Number>,filterMax: Array<Number>}}
+ * @returns {Array<Array<Number>>}
  */
 function getEmptyFilterRange() {
-  return {
-    filterMin: new Array(MAX_GPU_FILTERS).fill(0),
-    filterMax: new Array(MAX_GPU_FILTERS).fill(0)
-  };
+  return new Array(MAX_GPU_FILTERS).fill(0).map(d => [0, 0]);
```

Comment:
you can rewrite this as it follows:
```
Array(MAX_GPU_FILTERS).fill([0, 0]);
```

Reply:
the 4 [0, 0] has to be a different array.